### PR TITLE
feat(ConfirmerQueues): Scaffold confirmer queues

### LINKED ISSUE #187: Implement `DFCTransactionConfirmerQueue`
#### What would you like to be added:
As per title. This is similar to #183 . It might be possible to abstract away some parts of the confirmer queue, especially if we are going to support other chains in the future.

#### Why is this needed:
We need to confirm the transactions on DeFiChain for finality, before executing the follow up mint / burn transactions.

#### Partially Blocked by:
Number of confirmations for DFC is unknown, but this issue can still be started on though.

### PR DESCRIPTION
**Pull request by weiyuan95**
#### What this PR does / why we need it:
Scaffolds the confirmer queues we will need. These are empty shells - the respective confirmer implementation for DeFiChain and Ethereum will need to be filled in.

There is an example integration with the controller for the DeFiChain one.

#### Which issue(s) does this PR fixes?:

Partially fixes #187 and #183

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "test": "turbo run test",
     "test:hardhat": "turbo run test:hardhat --filter=./packages/smartcontracts",
     "dev": "turbo run dev --parallel",
-    "lint": "turbo run lint",
+    "lint": "turbo run lint -- --fix",
```
Comment (weiyuan95) [inline package.json:14]: We can `fix` the errors that the linter throws. Some of them, at least.

**Comment by weiyuan95** [comment]
Setting these as draft for now - we might not need this.

Also, should use the redis containers from `sticky` instead of rolling my own implementation.

**Comment by weiyuan95** [comment]
Setting this to draft for now - it might now be necessary given our current implementation.

File: apps/server/src/queue/DeFiChainTransactionConfirmerQueue.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    if ((await this.isTransactionConfirmed(transactionId)) === false) {
+      // Throw error here to fail job and let bull auto retry again after X seconds delay
+      throw new Error();
+    }
+  }
+
+  // TODO: Logic to check the number of confirmations
+  // eslint-disable-next-line  @typescript-eslint/no-unused-vars
+  async isTransactionConfirmed(transactionId: string): Promise<boolean> {
+    return true;
```
Comment (thedoublejay) [inline apps/server/src/queue/DeFiChainTransactionConfirmerQueue.ts:49]: The confirmer logic can be used on this service - https://github.com/WavesHQ/bridge/pull/250/files#diff-5eca400a67123ad45b8bf1e241fcc7c8160b8cadf4675c847b6288897124ab2bR57:~:text=waitForTxConfirmation

**Comment by thedoublejay** [comment]
@weiyuan95 can we close this

**Pull request closed without merging**